app/src/js/components/Errors/report.js

```diff
@@ -37,6 +38,7 @@ class ErrorReport extends React.Component {
         <div key={report}>
           <strong>Error:</strong>
           <ShowMoreOrLess text={report} truncate={this.props.truncate}></ShowMoreOrLess>
+          <CopyToClipboard text = {report}></CopyToClipboard>
```

Comment:
Another nit, but can we fix the spacing and make this a self closing tag?

```suggestion
<CopyToClipboard text={report} />
```

(And if you want to do the same thing for the `ShowMoreOrLess` component, I wouldn't complain :) )